mail-client/trojita/metadata.xml

```diff
@@ -14,7 +14,7 @@
 		<name>Gentoo KDE Project</name>
 	</maintainer>
 	<use>
-		<flag name="addressbook">Build <pkg>kde-apps/akonadi</pkg> addressbook plugin</flag>
+		<flag name="kde">Build <pkg>kde-apps/akonadi</pkg> addressbook plugin</flag>
```

Comment:
I disagree with the use of `kde` here, it does not describe what it does. Besides, users may want `kde`, but don't want `kde-apps/akonadi`.

USE `pim` on the other hand is already used in calligraplan, kraft, kde-apps-meta and simon for more or less the same purpose. It is not perfect either, but at least some precedent. Please also look at the metadata description of the given examples, you could use part of that for inspiration.